acceptance-tests/src/main/java/it/com/atlassian/bitbucket/jenkins/internal/util/BitbucketUtils.java

```diff
@@ -40,6 +45,16 @@ public class BitbucketUtils {
     public static final String REPO_ADMIN_PERMISSION = "REPO_ADMIN";
     public static final String REPO_SLUG = "rep_1";
 
+    public static final String PRIV_KEY = "-----BEGIN OPENSSH PRIVATE KEY-----\n" +
```

Comment:
I think I'd probably be prefer this be stored in a separate file and load it in rather than storing it as a raw string. It was also mean instead of having to copy the key out to a new file, you could just copy the existing file, making the `cerateED25519KeyPairOnDisk` method simpler. That said it means you need to read it in so pros and cons, happy either way.

Reply:
Yep, I agree and I think I am going to look into this next. I also want to implement @dkjellin 's suggestion of creating an artifact for the console output. For the sake of being able make a release asap, I think I am going to leave it and merge this. I will try to implement these things in another PR.